### PR TITLE
Redirect stdout from custom command to the log files

### PR DESCRIPTION
**Pull request by MYanello**

### CODE REVIEW

**Comment by MYanello** [comment]
For this feature request: https://github.com/arsenetar/dupeguru/issues/1007

File: core/app.py [hunk 1]
```diff
@@ -555,9 +555,13 @@ def invoke_custom_command(self):
                 # a workaround to make the damn thing work.
                 exepath, args = match.groups()
                 path, exename = op.split(exepath)
-                subprocess.Popen(exename + args, shell=True, cwd=path)
+                p = subprocess.Popen(exename + args, shell=True, cwd=path, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+                output = p.stdout.read()
+                logging.info("Custom command %s %s:\n %s", exename, args, output)
```
Comment (arsenetar) [inline core/app.py]: All other log entries are single line, I would prefer to keep these entries single line as well.

File: core/app.py [hunk 2]
```diff
@@ -555,9 +555,13 @@ def invoke_custom_command(self):
...
             else:
-                subprocess.Popen(dupe_cmd, shell=True)
+                p = subprocess.Popen(dupe_cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+                output = p.stdout.read()
+                logging.info("Custom command %s:\n %s", dupe_cmd, output)
```
Comment (arsenetar) [inline core/app.py]: All other log entries are single line, I would prefer to keep these entries single line as well.

**Review by arsenetar — CHANGES_REQUESTED** [review]
Thanks for the PR, I have one minor item I found during review.

File: core/app.py (hunk 1 above)
Comment (MYanello) [reply core/app.py]: Updated, thanks

File: core/app.py (hunk 2 above)
Comment (MYanello) [reply core/app.py]: Updated, thanks

**Comment by MYanello** [comment]
Thanks for the review and merge, out of curiosity, when are builds done? I'm having trouble creating the windows package (python38.dll not found) so was hoping to just wait for the next official build

**Comment by arsenetar** [comment]
Builds currently don't have a set schedule, mainly done based on the features / bugs fixed since last release.  The next release has an issue tracking its progress https://github.com/arsenetar/dupeguru/issues/1002.  Currently there are a few more minor items I am considering completing before building the next release.

**Pull request merged**